### PR TITLE
286/onto labnb

### PR DESCRIPTION
**Pull request by gnayuy**
refactored stimulus and labnotebook and added unit tests

### CODE REVIEW

File: tests/test_labnotebook.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from builtins import range
+import os
+import pytest
+import h5py
+import numpy as np
+from ipfx.dataset.labnotebook import LabNotebookReaderIgorNwb
+
+TEST_DATA_PATH = os.path.join(os.path.dirname(__file__), 'data')
+
+@pytest.mark.parametrize('NWB_file', ['H18.03.315.11.11.01.05.nwb'], indirect=True)
```
Comment (sgratiy) [inline tests/test_labnotebook.py]: have you checked whether this is a nwb2 file? We should probably test only nwb2 files.

File: tests/test_labnotebook.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
```
Comment (sgratiy) [inline tests/test_labnotebook.py]: we do not need `__future__` imports any longer

File: tests/test_labnotebook.py (hunk 1 above)
Comment (gnayuy) [reply tests/test_labnotebook.py]: Good point, have we uploaded any nwb2 file into tests/data folder? If not, I will upload one and update the test.

File: tests/test_labnotebook.py (hunk 2 above)
Comment (gnayuy) [reply tests/test_labnotebook.py]: Removed it.

File: tests/test_labnotebook.py (hunk 1 above)
Comment (gnayuy) [reply tests/test_labnotebook.py]: uploaded a NWB file 2.2.0 "Ctgf-T2A-dgCre;Ai14-495723.05.02.01-compressed-V2.nwb" to the tests/data folder

**Pull request closed without merging**